Jenkinsfile

```diff
@@ -21,45 +21,45 @@ node('vetsgov-general-purpose') {
   // setupStage
   dockerContainer = commonStages.setup()
 
-  stage('Lint|Security|Unit') {
-    if (params.cmsEnvBuildOverride != 'none') { return }
-
-    try {
-      parallel (
-        failFast: true,
-
-        lint: {
-          dockerContainer.inside(commonStages.DOCKER_ARGS) {
-            sh "cd /application && npm --no-color run lint"
-          }
-        },
-
-        // Check package.json for known vulnerabilities
-        security: {
-          retry(3) {
-            dockerContainer.inside(commonStages.DOCKER_ARGS) {
-              sh "cd /application && npm run security-check"
-            }
-          }
-        },
-
-        unit: {
-          dockerContainer.inside(commonStages.DOCKER_ARGS) {
-            sh "/cc-test-reporter before-build"
-            sh "cd /application && npm --no-color run test:unit -- --coverage"
-            sh "cd /application && /cc-test-reporter after-build -r fe4a84c212da79d7bb849d877649138a9ff0dbbef98e7a84881c97e1659a2e24"
-          }
-        }
-      )
-    } catch (error) {
-      commonStages.slackNotify()
-      throw error
-    } finally {
-      dir("vets-website") {
-        step([$class: 'JUnitResultArchiver', testResults: 'test-results.xml'])
-      }
-    }
-  }
+  // stage('Lint|Security|Unit') {
+  //   if (params.cmsEnvBuildOverride != 'none') { return }
+
+  //   try {
+  //     parallel (
+  //       failFast: true,
+
+  //       lint: {
+  //         dockerContainer.inside(commonStages.DOCKER_ARGS) {
+  //           sh "cd /application && npm --no-color run lint"
+  //         }
+  //       },
+
+  //       // Check package.json for known vulnerabilities
+  //       security: {
+  //         retry(3) {
+  //           dockerContainer.inside(commonStages.DOCKER_ARGS) {
+  //             sh "cd /application && npm run security-check"
+  //           }
+  //         }
+  //       },
+
+  //       unit: {
+  //         dockerContainer.inside(commonStages.DOCKER_ARGS) {
+  //           sh "/cc-test-reporter before-build"
+  //           sh "cd /application && npm --no-color run test:unit -- --coverage"
+  //           sh "cd /application && /cc-test-reporter after-build -r fe4a84c212da79d7bb849d877649138a9ff0dbbef98e7a84881c97e1659a2e24"
+  //         }
+  //       }
+  //     )
+  //   } catch (error) {
+  //     commonStages.slackNotify()
+  //     throw error
+  //   } finally {
+  //     dir("vets-website") {
+  //       step([$class: 'JUnitResultArchiver', testResults: 'test-results.xml'])
+  //     }
+  //   }
+  // }
```

Comment:
Can this commented-out code be removed before merging?

Reply:
Yeah, I'm refactoring the PR to parallelize the tests further. I'll take this out once I finish testing.